addons/html_editor/static/src/main/link/link_popover.js

```diff
@@ -63,12 +77,99 @@ export class LinkPopover extends Component {
             type:
                 this.props.type ||
                 this.props.linkElement.className
-                    .match(/btn(-[a-z0-9_-]*)(primary|secondary)/)
+                    .match(/btn(-[a-z0-9_-]*)(primary|secondary|custom)/)
                     ?.pop() ||
                 "",
+            linkTarget: this.props.linkElement.target === "_blank" ? "_blank" : "",
+            buttonSize: this.props.linkElement.className.match(/btn-(sm|lg)/)?.[1] || "",
+            customBorderSize: computedStyle.borderWidth.replace("px", "") || "1",
+            customBorderStyle: computedStyle.borderStyle || "solid",
             isImage: this.props.isImage,
         });
 
+        this.customTextColorState = useState({
+            selectedColor: computedStyle.color || "#714B67",
+            defaultTab: "solid",
+        });
+        this.customTextResetPreviewColor = this.customTextColorState.selectedColor;
+        this.customFillColorState = useState({
+            selectedColor: computedStyle.backgroundColor || "#ffffff",
```

Comment:
same as above. `DEFAULT_FILL_COLOR`?

btw, considering this is white, does it play well on dark mode?

Reply:
It's not a problem for dark mode, as it's a custom button that will use the default text and fill color, which work together and will be "hard coded" in the style of the button.